pkg/proxy/apis/kubeproxyconfig/v1alpha1/doc.go

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// +k8s:deepcopy-gen=package,register
+// +k8s:conversion-gen=k8s.io/kubernetes/pkg/proxy/apis/kubeproxyconfig
+// +k8s:openapi-gen=true
```

Comment:
I don't think this has any effect, because this api group is not listed in https://github.com/kubernetes/kubernetes/blob/618b705a4bddc6a6938d950135e05496a620f454/hack/lib/init.sh#L53-L81, nor is it served by the apiserver. Could you please confirm @deads2k @sttts?

Reply:
> , nor is it served by the apiserver. Could you please confirm @deads2k @sttts?

Because it isn't served, we don't care if there is any open-api generated.

Reply:
Although @smarterclayton was asking if there was some way for `kubectl explain` to explain the config structs, but the only way to do that afaik is if it's in swagger....

Reply:
If both here and https://github.com/kubernetes/kubernetes/pull/53645#discussion_r148271470 are ok as-is anyway, I'd like this PR merged asap per @danehans concern :)